src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -356,6 +358,48 @@ export function transform(formConfig, form) {
     );
   };
 
+  const addRequiredDescriptionsToDisabilitiesBDD = formData => {
+    if (!isBDDForm || !formData.newDisabilities) {
+      return formData;
+    }
+
+    const newDisabilitiesWithRequiredDescriptions = formData.newDisabilities.map(
+      disability => {
+        const disabilityDescription = {};
+
+        switch (disability.cause) {
+          case causeTypes.NEW:
+            disabilityDescription.primaryDescription =
+              'This disability/symptom is related to my military service.';
+            break;
+          case causeTypes.SECONDARY:
+            disabilityDescription.causedByDisabilityDescription =
+              'This disability/symptom was caused by another condition.';
+            break;
+          case causeTypes.WORSENED:
+            disabilityDescription.worsenedDescription =
+              'This pre-existing disability/symptom was worsened by military service.';
+            disabilityDescription.worsenedEffects =
+              'This pre-existing disability/symptom was worsened by military service.';
+            break;
+          case causeTypes.VA:
+            disabilityDescription.vaMistreatmentDescription =
+              'This disability/symptom was caused by an injury or event that happened while I was receiving VA care.';
```

Comment:
I am going to push another commit to change all of these strings to constants in our constants file. Also, the worsened descriptions are pending changes based on a response from Barry/Kevin.